mne/realtime/base_client.py

```diff
@@ -3,13 +3,13 @@
 #
 # License: BSD (3-clause)
 
-import copy
+import threading
```

Comment:
if you have a better solution, i welcome it. here we are just extracting away from our existing code and wanted to have a lightweight base class object that would be work across our existing clients. I wasn't aware of the problems with multithreading